.gitignore

```diff
@@ -181,6 +204,9 @@ publish/
 PublishScripts/
 
 # NuGet Packages
+*.nupkg
```

Comment:
We have a package in our source code `SevenZipSharp.1.0.2.nupkg` so it's best to avoid this change.

Reply:
Why isn't this on NuGet? Isn't this deprecated?

Reply:
We use [customized version of SevenZipSharp](https://github.com/files-community/SevenZipSharp).

Reply:
> We use [customized version of SevenZipSharp](https://github.com/files-community/SevenZipSharp).

Oh I see, I'll remove it from the ignore - why don't you publish this library to NuGet?

Reply:
Wouldn't it be easier to update then?